### PR TITLE
feat: support for swaync

### PR DESCRIPTION
**Pull request by Gl00ria**
Hi, @prasanthrangan @kRHYME7 PR's ready. **NOTE:** it's a WIP till the integration with 'theme switching' is complete

## Description:
Implementation for [SwayNC](https://github.com/ErikReider/SwayNotificationCenter?tab=readme-ov-file) in order to replace [Dunst](https://github.com/dunst-project/dunst).

## Type of change:

Please put an `x` in the boxes that apply:

- [ ] **Bug fix** (non-breaking change which fixes an issue)
- [x] **New feature** (non-breaking change which adds functionality)
- [x] **Breaking change** (fix or feature that would cause existing functionality to not work as expected)
- [ ] **Documentation update** (non-breaking change; modified files are limited to the documentations)
- [ ] **Technical debt** (a code change that does not fix a bug or add a feature but makes something clearer for devs)
- [ ] **Other** (provide details below)

## Checklist

Please put an `x` in the boxes that apply:

- [x] I have read the [CONTRIBUTING](https://github.com/prasanthrangan/hyprdots/blob/main/CONTRIBUTING.md) document.
- [x] My code follows the code style of this project.
- [x] My commit message follows the [commit guidelines](https://github.com/prasanthrangan/hyprdots/blob/main/CONTRIBUTING.md#git-commit-messages).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added a changelog entry.
- [x] I have added necessary comments/documentation to my code.
- [ ] I have added tests to cover my changes.
- [x] I have tested my code locally and it works as expected.
- [ ] All new and existing tests passed.

## Screenshots
Mentioned in #1023 

## Additional context
1. What font you want to be used?
2. What is the height & width standard you want? I have a big screen, so I couldn't decide :(
3. What are the functionality you want for each button?
4. How to edit the scripts to switch the theme correctly?

### CODE REVIEW

**Comment by Enigma1309** [comment]
lgtm but can you add wall-bash support?

File: Configs/.config/swaync/config.json [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+          "label": "  ",
+          "command": "blueman-manager"
+        },
+        {
+          "label": " 󰃛 ",
+          "command": "hyprshade toggle bluefilter"
+        },
+        {
+          "label": "  ",
+          "command": "sh -c ~/.config/hypr/scripts/logoutlaunch.sh"
```
Comment (Enigma1309) [inline Configs/.config/swaync/config.json:80]: the script dir is changed to `.local/share/bin/`

File: Configs/.config/swaync/config.json [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+          "label": " 󱛆  ",
+          "command": "nm-connection-editor"
+        },
+        {
+          "label": "  ",
+          "command": "blueman-manager"
+        },
+        {
+          "label": " 󰃛 ",
+          "command": "hyprshade toggle bluefilter"
```
Comment (Enigma1309) [inline Configs/.config/swaync/config.json:76]: the command should be `hyprshade toggle blue-light-filter`

File: Configs/.config/swaync/config.json (hunk 1 above)
Comment (Gl00ria) [reply Configs/.config/swaync/config.json:80]: I'm aware of that, but as I mentioned in the **Additional context**:

> What are the functionality you want for each button

So I left it as it is for further changes

File: Configs/.config/swaync/config.json (hunk 2 above)
Comment (Gl00ria) [reply Configs/.config/swaync/config.json:76]: I have a custom 'shader' file called 'bluefilter'

**Comment by Gl00ria** [comment]
> lgtm but can you add wall-bash support?

how about [themeswitch.sh](https://github.com/prasanthrangan/hyprdots/blob/main/Configs/.local/share/bin/themeswitch.sh)?

File: Configs/.config/swaync/config.json (hunk 2 above)
Comment (Gl00ria) [reply Configs/.config/swaync/config.json:76]: I have a custom shader file called 'bluefilter'

File: Configs/.config/hypr/keybindings.conf [hunk 3]
```diff
@@ -63,6 +63,9 @@ bind = $mainMod+Ctrl, P, exec, $scrPath/screenshot.sh sf # partial screenshot ca
 bind = $mainMod+Alt, P, exec, $scrPath/screenshot.sh m # monitor screenshot capture
 bind = , Print, exec, $scrPath/screenshot.sh p # all monitors screenshot capture
 
+# notification center
+bind = $mainMod, N, exe, swaync-client -t
```
[the remark this answers is not in the document]
Comment (Gl00ria) [reply Configs/.config/hypr/keybindings.conf]: fixed and moved it to the app category

**Comment by abenezerw** [comment]
image
i dont know why its oversized nor detecting the style. maybe something wrong with me themes placement?

**Comment by rubiin** [comment]
That clearly a bug on sizing . I was thinking of something like this
https://www.reddit.com/media?url=https%3A%2F%2Fpreview.redd.it%2Fswaync-dynamic-colors-v0-vmvn5a5taylc1.png%3Fwidth%3D1080%26crop%3Dsmart%26auto%3Dwebp%26s%3D83c94967ba40c6c782657c5b5cf6d3b9e24402de

**Comment by rubiin** [comment]
Theming doesnt seem to be a issue for now . The only issue is with sizing which is bound to cause issues with users

**Comment by rubiin** [comment]
@Gl00ria you can take some inpirations for swaync https://github.com/AXWTV/Hyprland-DotFiles/blob/main/config/waybar/modules

**Pull request closed without merging**